libs/core/src/lib/resizable-card-layout/resizable-card-item/resizable-card-item.component.ts

```diff
@@ -0,0 +1,588 @@
+import {
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    EventEmitter,
+    HostBinding,
+    HostListener,
+    Input,
+    Output,
+    ViewEncapsulation
+} from '@angular/core';
+import { FocusableOption } from '@angular/cdk/a11y';
+
+export type ResizeDirection = 'vertical' | 'horizontal' | 'both';
+
+// Card resizes in step of fixed values. values are in pixel
+export const horizontalResizeStep = 320;
+export const verticalResizeStep = 16;
+export const gap = 16;
+export const horizontalResizeOffset = 160;
+export const verticalResizeOffset = 10;
+export type CardState = 1 | 0 | -1;
+
+let cardRank = 1;
```

Comment:
please add a comment about `cardRank` - what it is and what value it accepts.

Reply:
added typedoc